packages/api-openfin/src/window.ts

```diff
@@ -38,10 +38,26 @@ const windowStates = {
 const DEFAULT_WIDTH = 800;
 const DEFAULT_HEIGHT = 600;
 
+const isUrlPattern = /^https?:\/\//i;
+
+const guid = () => {
+  const s4 = () => {
+    return Math.floor((1 + Math.random()) * 0x10000)
+      .toString(16)
+      .substring(1);
+  }
+  return s4() + s4() + '-' + s4() + '-' + s4() + '-' +
+    s4() + '-' + s4() + s4() + s4();
+};
+
+const defaultOptions = {
+  name: guid(),
```

Comment:
As the `guid` function is invoked when the `defaultOptions` object is being constructed, doesn't this mean that all windows will be given the same name?